src/pages/data-service-details-page/data-service-details-page.jsx

```diff
@@ -331,10 +331,12 @@ const renderStickyMenu = (apiItem, informationModels, datasets) => {
     });
   }
   if (
-    apiItem?.contactPoint[0]?.hasURL ||
-    apiItem?.contactPoint[0]?.organizationName ||
-    apiItem?.contactPoint[0]?.email ||
-    apiItem?.contactPoint[0]?.hasTelephone
+    Array.isArray(apiItem?.contactPoint) &&
+    apiItem.contactPoint.length > 0 &&
+    (apiItem.contactPoint[0].hasURL ||
```

Comment:
Er det ikke et potensial for at det kommer flere contactPoints? Burde ikke alle sjekkes om de har interessante verdier?